src/components/generic/components/appeal_card.js

```diff
@@ -18,7 +18,7 @@ function Appeal_card(props) {
           <p className="text-[1.6rem] font-normal tracking-[-0.4px] leading-[2.4rem] text-body">Every single day, women around the world have to walk miles to collect water for their household. Installing a hand pump brings this basic human right closer to home.</p>
           <div className="flex gap-4 items-center mt-12">
             <div className="w-1/5">
-              <Circular_progress_bar_3 percentage="50" fontSize="2" />
+              <CircularProgressbarComponent percentage={60} fontSize="2" width="7rem" height="7rem"/>
```

Comment:
It would be bettet if we just pass the style props from here that include the fontsize width snd height and use in other component
So that it will be generic enough
Like 
style= {{ fontSize: 2, width: 7rem, height: 7rem}}